images/helm-operator/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM centos:7 as build
 RUN yum -y update && yum clean all
 
 RUN mkdir -p /go && chmod -R 777 /go && \
-    yum -y install epel-release && yum -y install hg git golang glide && yum group install -y "Development Tools"
+    yum -y install epel-release && yum -y install protobuf hg git golang glide make
```

Comment:
I thought kubectl was one too?

Reply:
ah, it looks like I wasn't looking at `epel`. There's a 1.7.5 copy of `kubectl` in there.

I don't see it in brew, which is worth noting.

Reply:
1.5 is probably too old